packages/node-core/src/indexer/store.service.ts

```diff
@@ -191,10 +191,19 @@ export class StoreService {
 
       /* These SQL queries are to allow hot-schema reload on query service */
       extraQueries.push(createSchemaTriggerFunction(schema));
-      extraQueries.push(createSchemaTrigger(schema));
+      const schemaTriggerName = makeTriggerName(schema, '_metadata', 'schema');
+
+      const schemaNotifyTrigger = await this.sequelize.query(getNotifyTriggers(), {
```

Comment:
maybe rename `getNotifyTriggers()` to `getTriggers()`, since both `notify` and `schema` trigger are using it